packages/traceloop-sdk/examples/dataset_attachments_example.py

```
@@ -0,0 +1,357 @@
+    try:
+        # Run examples
+        example_external_attachments()
+        example_file_uploads()
+        example_in_memory_attachments()
+        example_mixed_attachments()
+
+        print("\n" + "=" * 60)
+        print("All examples completed successfully!")
+        print("=" * 60)
+
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)